common/lib/dependabot/update_checkers/base.rb

```diff
@@ -293,6 +293,8 @@ def version_from_requirements
           reject { |req_array| req_array.first.start_with?("<") }.
           map(&:last).
           max
+      rescue requirement_class::BadRequirementError
```

Comment:
Should we rescue *any* `BadRequirementError`? I'd expect some more specific check on `latest` being set, but maybe nothing else is possible in this case?

But looking at the code, it seems like this would now no longer raise:

```json
"dependencies": {
  "etag" : "some-invalid-string-that-npm-doesnt-know"
}
```

Reply:
Oh yes good catch! Will have a think about how to handle this better in the npm update checker 👍